scripts/generateAssetData/cosmos/index.ts

```diff
@@ -0,0 +1,96 @@
+import { cosmosChainId } from '@shapeshiftoss/caip'
+import axios from 'axios'
+import type { Asset } from 'lib/asset-service'
+
+import { atom } from '../baseAssets'
+import { colorMap } from '../colorMap'
+import type { IdenticonOptions } from '../generateAssetIcon/generateAssetIcon'
+import { getRenderedIdenticonBase64 } from '../generateAssetIcon/generateAssetIcon'
+
+type CosmosAsset = {
+  denom: string
+  type: string
+  origin_chain: string
+  origin_denom: string
+  origin_type: string
+  symbol: string
+  decimals: number
+  enable?: boolean
+  path?: string
+  channel?: string
+  port?: string
+  counter_party?: {
+    channel: string
+    port: string
+    denom: string
+  }
+  description?: string
+  image?: string
+  coinGeckoId: string
+}
+
+export const getAssets = async (): Promise<Asset[]> => {
```

Comment:
should add this to `generateColorMap`, generate the color map and then regen the asset data so not everything is #FFFFFF

Reply:
https://github.com/shapeshift/web/pull/5141/commits/915666979abf78925d1ebef32dbee4f1e41ed388 and https://github.com/shapeshift/web/pull/5141/commits/bf3cfdacbd31ef017123ecd15fbe601d47a2f20d